packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/InAppPurchasePlugin.swift

```diff
@@ -27,6 +27,10 @@ public class InAppPurchasePlugin: NSObject, FlutterPlugin, InAppPurchaseAPI {
   // This property is optional, as it requires self to exist to be initialized.
   public var paymentQueueHandler: FLTPaymentQueueHandlerProtocol?
 
+  // This should be an Task, but Task is on available >= iOS 13
+  var updateListenerTask: Any?
+  var transactionDelegate: InAppPurchase2CallbackAPI? = nil
```

Comment:
can you call it `transactionCallbackAPI`? I intended to say use "Delegate" if it's a separate class

Reply:
done